config-path.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const userHome = require('user-home');
+
+const env = process.env;
+const name = 'js-v8flags';
+
+function macos () {
+	const library = path.join(userHome, 'Library');
+	return path.join(library, 'Caches', name);
+}
+
+function windows () {
+	const appData = env.LOCALAPPDATA || path.join(userHome, 'AppData', 'Local');
+	return path.join(appData, name, 'Cache');
+}
+
+// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+function linux () {
+	const username = path.basename(userHome);
+	return path.join(env.XDG_CACHE_HOME || path.join(userHome, '.cache'), name);
+}
+
+if (process.platform === 'darwin') {
```

Comment:
There's enough in this library that makes it unlikely to be bundled that I don't think the extra overhead is worth it to care about static analysis.

@Siilwyn It looks like it might be a problem with the removal of items from the cache, but I'm not 100% sure.  You can use the `nave` tool to switch between node versions and test locally.